Sprint-1/fix/median.js

```diff
@@ -3,9 +3,13 @@
 // If you're in the Sprint-1 directory, you can run `npm test -- fix` to run the tests in the fix directory
 
 function calculateMedian(list) {
+  list.sort((a, b) => a - b);
+  console.log([list].join(""));
```

Comment:
Array is passed to a function ***by reference***  (instead of ***by value***). This implies when an array is passed to a function via parameter, the array is **shared** by the caller and the callee (the function being called).

To avoid modifying the caller's array, a common practice is to make a clone of the array (that's passed as a parameter) and modify the clone in the callee instead.

Line 6: `.sort()` can modify the content in `list`, so a better practice is to make a clone of `list`, and sort the cloned array.

Line 7: It is never too early to develop the habit to keep the code clean by removing 
(or at least commenting out) unused code or debugging code.

Reply:
Hi @cjyuan ,

Thank you for the helpful feedback! I completely agree with your points.

1-Cloning the Array: You're right that modifying the original array can cause unexpected side effects. I’ll make sure to clone the array before sorting it, which will keep the original list intact.

2-Removing Debugging Code: I also appreciate your reminder about cleaning up unused code. I’ll remove the console.log statement to keep the code neat and maintain best practices.

Thanks again for your suggestions 🙏they’ll definitely improve the code.😊